pgtest.sql

```diff
@@ -309,15 +306,24 @@ $$ LANGUAGE plpgsql
   SECURITY DEFINER
   SET search_path=pgtest, pg_temp;
 
+CREATE OR REPLACE FUNCTION pgtest.mock_function_id(s_original_function_schema_name VARCHAR, s_original_function_name VARCHAR, s_function_argument_types VARCHAR[]) RETURNS varchar AS $$
+BEGIN
+  RETURN 'pgtest_mock_' || md5(s_original_function_schema_name || s_original_function_name || array_to_string(s_function_argument_types, ''));
```

Comment:
Separate s_original_function_schema_name, s_original_function_name and s_function_argument_types with some delimiter (for example "."). Otherwise it is possible to have a scenario where schema named A with a function BB and schema AB with a function B end up with same mock_id. There should be test to check this scenario.

Also what would you think if the function will be called "get_mock_id"?

Reply:
done